Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,627 @@
+
+A link here.(https://technet.microsoft.com/en-us/library/cc726016(v=ws.10).aspx)
+
+  - The actual creation of computer and user accounts is done via the "Active Domain Users And Computers" utility on the ADC.
+  - Service Principals are created using the setspn.exe command line utility. 
+  - Information about the active domain state can be gotten from via ldapsearch (Windows or Cleint side) or throgh powershell,
+    via the get-ADComputer() and get-ADUser cmdlets.
+
+### Redhat/Centos Linux 7 (all versions)  (Should also apply to Fedora 24 and later).
+
+Redhat has an excellent write up on setting up an active domain client 
+here.
+(https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Linux_Domain_Identity_Authentication_and_Policy_Guide/index.html)
+which describes the process with all of the possible considerations.  Some might find it a bit long, but it is recommended
+reading.
+
+####  Required Packages:
+
+We recommend you use realmd to join. The required packages in that care are 
+```
+    sudo yum install sssd sssd-client krb5-workstation samba openldap-clients open-ssl authconfig realmd oddjob oddjob-mkhomedir adcli 
+```
+
+#### Setup after package install
+
+
+Tasks:
+
+  - Hostname - edit /etc/hostname to create an fqdn with a short name (the hostname proper) of 15 characters or less.
+
+  - DNS - point at the ADC as the dns name server by editing the source /etc/resolv.conf. Optionally add an entry in /etc/hosts.
+    pointing at the ADC to ensure that is available.
+
+    Note:  If you are using Network Manager, just modifying /etc/resolv.conf will not me sufficient in the long run as 
+    Network Manager clobbers /etc/resolv.conf on every reboot. If so, modify the network manager configuration files 
+    using the redhat-network-configuration tool or as needed. 
+    *but ensure you have a fully qualified domain name by default before proceeding*
+
+  - NTP is necessary. It should point at the ADC plus a number of pool servers. Authentication will fail more or less 
+    mysteriously if the time is out of sync by more than five minutes between the kdc, client and host its trying to 
+    communicate with.
+
+  - Kerberos: Add /etc/krb5.conf.  We recommend that you keep a master krb5.conf and just copy it to each machine. The 
+    contents of /etc/krb5.conf are not host sensitive, though they are of course, unique to the domain.  Realmd and 
+    other authconfigure tools depend on at least a minimal /etc/krb5.conf in order to function.
+
+  - Authconfig will setup the configuration files /etc/nsswitch, /etc/smb.conf, /etc/pam.d/*, /etc/sssd/sssd.conf.
+
+  - Realmd will complete the setup.
+
+  - it will be necessary to add an additional service principal to the  machine account after the join is complete.
+
+
+#### hostname 15 char or less and fully qualified
```

Comment:
Is this what you meant? At least capitalize 'h' of hostname if it's supposed to be a title.